client/src/components/shared/ListingModal.component.js

```diff
@@ -28,6 +28,10 @@ export default class ListingModal extends Component {
     this.props.handleClose();
   }
 
+  listingFromId(id) {
```

Comment:
This is a method that takes a given value and returns it, and it's not used... can remove?